src/omv/boards/ARDUINO_GIGA/omv_boardconfig.h

```diff
@@ -37,6 +37,15 @@
 #define OMV_MDMA_CHANNEL_JPEG_IN            (7) // in has a lower pri than out
 #define OMV_MDMA_CHANNEL_JPEG_OUT           (6) // out has a higher pri than in
 
+// OV5640 sensor settings
```

Comment:
I will test this later today.

Reply:
> Can you confirm the above settings work or not? Using 12.5MHz is not an integer multiple of 480 MHz. So, the clock is really going down to 12 MHz. You should write 12000000 then instead of 12500000.
> 
> Finally, the final output clock of the camera should is 24MHz/3=8MHz * OMV_OV5640_PLL_CTRL2 / 10 = 80 MHz. The STM32 should handle that speed. If not, please just only lower the OMV_OV5640_PLL_CTRL2 value. You should aim to get it working at the highest possible value.
> 
> Right now your PR is running the camera PCLK at 12MHz/3=4MHz * 126 = 504 MHz / 10 = 50.4MHz. You should be able to go faster.
> 
> Finally, you don't need the REV_Y defines if the check one is 0.

Unfortunately it seems the clock is not stable

Reply:
I have ended up reverting the change as the FPS is actually higher with the portenta like settings. I have however removed the non necessary definitions.

Reply:
> 0x3E

Should I also rewrite the XCLK FREQ or leave it as 24MHZ?

Reply:
12 MHz is fine, but, then use 7C (3E  * 2).

E.g. either MCLK of 12 or 24 MHz with PLL_CTRL2 of 0x3E or 0x7C.

Either does 50 MHz oout.

Reply:
Try to get to 52.5 MHz then. The Portenta runs the camera at this. Note that the Portenta runs at 400 MHz. So, 12.5MHz is an integer divisor of 400. The 12.5MHz is then divided by 3 by the camera for 4.1666 MHz feed to it's PLL which then multiplies it up to 525 MHz and the PCLK output is 52.5 MHz.

So... 0x84. Find the highest even number you can get to.

Reply:
Seems like 0x90 is the highest properly stable value, I have now pushed this change on the side of the PR.